### PR TITLE
Added swap case program and removed unexpected expression part

### PR DESCRIPTION
**Pull request by mayur200**
### **Describe your change:**
* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?
* [ ] I have added an algorithm for swap case in string folder
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I have fixed unexpected expression part from 2 algorithm.
* [ ] All functions and variable names follow Python naming conventions.

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: sorts/cocktail_shaker_sort.py [hunk 1]
```diff
@@ -42,4 +42,4 @@ def cocktail_shaker_sort(unsorted: list) -> list:
     doctest.testmod()
     user_input = input("Enter numbers separated by a comma:\n").strip()
     unsorted = [int(item) for item in user_input.split(",")]
-    print(f"{cocktail_shaker_sort(unsorted) = }")
+    print(f"{cocktail_shaker_sort(unsorted) }")
```
Comment (cclauss) [inline sorts/cocktail_shaker_sort.py:45]: Why break this?

File: strings/naive_string_search.py [hunk 2]
```diff
@@ -39,4 +39,4 @@ def naive_pattern_search(s: str, pattern: str) -> list:
 
 if __name__ == "__main__":
     assert naive_pattern_search("ABCDEFG", "DE") == [3]
-    print(f"{naive_pattern_search('ABAAABCDBBABCDDEBCABC', 'ABC') = }")
+    print(f"{naive_pattern_search('ABAAABCDBBABCDDEBCABC', 'ABC')}")
```
Comment (cclauss) [inline strings/naive_string_search.py:42]: Why break this?

File: strings/swap_case.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+def swap_case(sentence):
+    """
+    This function will convert all lowercase letters to uppercase letters and vice versa.
+    >>>swap_case('Algorithm.Python@89')
+    aLGORITHM.pYTHON@89
+    """
+    if len(sentence) < 1001:
+        newstring = ''
+        for word in sentence:
```
Comment (cclauss) [inline strings/swap_case.py:32]: These are not words.  They are char.

File: strings/swap_case.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            if word.isupper() == True:
+                newstring += word.lower()
+            if word.islower() == True:
+                newstring += word.upper()
+            if regexp.search(word):
+                newstring += word
+
+    else:
+        raise Exception("Charater length should be between 1 and 1000")
```
Comment (cclauss) [inline strings/swap_case.py:41]: Why?

File: strings/swap_case.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+Constraints: Length of string should between 1 and 1000
+
+
+'''
+import re
+'''
+This re.compile() function saves the pattern from 'a' to 'z' and 'A' to 'Z' into 'regexp' variable
+'''
+regexp = re.compile('[^a-zA-Z]+')
```
Comment (cclauss) [inline strings/swap_case.py:21]: This is never used.

File: strings/swap_case.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+
+def swap_case(sentence):
+    """
+    This function will convert all lowercase letters to uppercase letters and vice versa.
+    >>>swap_case('Algorithm.Python@89')
```
Comment (cclauss) [inline strings/swap_case.py:27]:
```suggestion
    >>> swap_case('Algorithm.Python@89')
```
Without the space, the test is never run.

File: strings/swap_case.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+User will give input and then program will perform swap cases.
+
+In other words, convert all lowercase letters to uppercase letters and vice versa.
+For example:
+1)>>>Please input sentence: Algorithm.Python@89
+  aLGORITHM.pYTHON@89
+2)>>>Please input sentence: github.com/mayur200
+  GITHUB.COM/MAYUR200
+
+Constraints: Length of string should between 1 and 1000
```
Comment (cclauss) [inline strings/swap_case.py:13]: Why?

**Review by cclauss — CHANGES_REQUESTED** [review]

**Comment by cclauss** [comment]
All tests are failing.

File: strings/swap_case.py (hunk 4 above)
Comment (mayur200) [reply strings/swap_case.py:41]: I have raise an exception and its readable.

File: strings/naive_string_search.py (hunk 2 above)
Comment (mayur200) [reply strings/naive_string_search.py:42]: I was getting error in pycharm 'unexpected expression part' before running a program.

File: sorts/cocktail_shaker_sort.py (hunk 1 above)
Comment (mayur200) [reply sorts/cocktail_shaker_sort.py:45]: I was getting error in pycharm 'unexpected expression part' before running a program.

File: strings/swap_case.py (hunk 6 above)
Comment (mayur200) [reply strings/swap_case.py:27]: I will fix it

File: strings/swap_case.py (hunk 7 above)
Comment (mayur200) [reply strings/swap_case.py:13]: Its example for problem statement. If someone failed to understand the problem. This example is self explanatory

File: strings/swap_case.py (hunk 3 above)
Comment (mayur200) [reply strings/swap_case.py:32]: ok. I will fix it

File: strings/swap_case.py (hunk 4 above)
Comment (cclauss) [reply strings/swap_case.py:41]: No.  I mean why create a 1000 character limit?  Just process the string no matter how long it is.

File: strings/swap_case.py (hunk 7 above)
Comment (cclauss) [reply strings/swap_case.py:13]: No. I mean why create a 1000 character limit? Just process the string no matter how long it is.

Comment (mayur200) [reply strings/swap_case.py:13]: Ok. I will fix it.

File: strings/naive_string_search.py (hunk 2 above)
Comment (cclauss) [reply strings/naive_string_search.py:42]: Pycharm is not upgraded to Python 3.8 and 3.9 which support this functionality.

File: sorts/cocktail_shaker_sort.py (hunk 1 above)
Comment (mayur200) [reply sorts/cocktail_shaker_sort.py]: I have fixed it again and I have commited changes. Sorry for the inconvenience

File: strings/naive_string_search.py (hunk 2 above)
Comment (mayur200) [reply strings/naive_string_search.py]: I have fixed this

File: strings/swap_case.py (hunk 3 above)
Comment (mayur200) [reply strings/swap_case.py]: I have fixed it and commit the changes

File: strings/swap_case.py (hunk 4 above)
Comment (mayur200) [reply strings/swap_case.py]: I have removed condition

File: strings/swap_case.py (hunk 5 above)
Comment (mayur200) [reply strings/swap_case.py:21]: I have used _regexp_   `if regexp.search(char):
            newstring += char` here

File: strings/swap_case.py (hunk 6 above)
Comment (mayur200) [reply strings/swap_case.py]: I have added space. I have commited changes

File: strings/swap_case.py (hunk 7 above)
Comment (mayur200) [reply strings/swap_case.py]: I have fixed it and I have pushed changes

File: strings/swap_case.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+def swap_case(sentence):
+    """
+    This function will convert all lowercase letters to uppercase letters and vice versa.
+
+    >>> swap_case('Algorithm.Python@89')
+    aLGORITHM.pYTHON@89
+    """
+    new_string = ""
+    for char in sentence:
+        if char.isupper() == True:
```
Comment (cclauss) [inline strings/swap_case.py:29]:
```suggestion
        if char.isupper():
```

File: strings/swap_case.py [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+            new_string += char.lower()
+        if char.islower() == True:
```
Comment (cclauss) [inline strings/swap_case.py:31]:
```suggestion
        if char.islower():
```

**Comment by mayur200** [comment]
Why my test is failed again??
I am confused I don't know why its failed my program gives me perfect output

**Comment by cclauss** [comment]
https://travis-ci.com/github/TheAlgorithms/Python/jobs/399797450#L775

File: strings/swap_case.py [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+            new_string += char.upper()
+        if regexp.search(char) == True:
+            new_string += char
+
+    return new_string
+
+
+if __name__ == '__main__':
```
Comment (cclauss) [inline strings/swap_case.py:39]:
```suggestion
if __name__ == "__main__":
```

File: strings/swap_case.py [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+# This re.compile() function saves the pattern from 'a' to 'z' and 'A' to 'Z' into 'regexp' variable
+regexp = re.compile("[^a-zA-Z]+")
+
+
+def swap_case(sentence):
+    """
+    This function will convert all lowercase letters to uppercase letters and vice versa.
+
+    >>> swap_case('Algorithm.Python@89')
+    aLGORITHM.pYTHON@89
```
Comment (cclauss) [inline strings/swap_case.py:25]:
```suggestion
    'aLGORITHMpYTHON```

File: strings/swap_case.py [hunk 12]
```diff
@@ -0,0 +1,42 @@
+"""
+This algorithm helps you to swap cases.
+
+User will give input and then program will perform swap cases.
+
+In other words, convert all lowercase letters to uppercase letters and vice versa.
+For example:
+1)>>> Please input sentence: Algorithm.Python@89
```
Comment (cclauss) [inline strings/swap_case.py:8]:
```suggestion
1. Please input sentence: Algorithm.Python@89
```

File: strings/swap_case.py [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+  aLGORITHM.pYTHON@89
+2)>>> Please input sentence: github.com/mayur200
```
Comment (cclauss) [inline strings/swap_case.py:10]:
```suggestion
2. Please input sentence: github.com/mayur200
```

File: strings/swap_case.py [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+  GITHUB.COM/MAYUR200
+
+"""
+import re
+
+# This re.compile() function saves the pattern from 'a' to 'z' and 'A' to 'Z' into 'regexp' variable
```
Comment (cclauss) [inline strings/swap_case.py:16]:
```suggestion
# This re.compile() function saves the pattern from 'a' to 'z' and 'A' to 'Z'
# into 'regexp' variable
```

File: strings/swap_case.py [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+regexp = re.compile("[^a-zA-Z]+")
+
+
+def swap_case(sentence):
+    """
+    This function will convert all lowercase letters to uppercase letters and vice versa.
```
Comment (cclauss) [inline strings/swap_case.py:22]:
```suggestion
    This function will convert all lowercase letters to uppercase letters
    and vice versa.
```

File: strings/swap_case.py [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+    >>> swap_case('Algorithm.Python@89')
+    aLGORITHM.pYTHON@89
+    """
+    new_string = ""
+    for char in sentence:
+        if char.isupper() == True:
+            new_string += char.lower()
+        if char.islower() == True:
+            new_string += char.upper()
+        if regexp.search(char) == True:
```
Comment (cclauss) [inline strings/swap_case.py:33]:
```suggestion
        if regexp.search(char):
```

File: strings/swap_case.py [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+            new_string += char
+
+    return new_string
+
+
+if __name__ == '__main__':
+    s = input("Please input sentence:")
+    result = swap_case(s)
+    print(result)
```
Comment (cclauss) [inline strings/swap_case.py:42]:
```suggestion
    print(swap_case(input("Please input sentence:")))
```

**Review by cclauss — APPROVED** [review]

**Pull request merged**